src/plugins/intel_cpu/src/emitters/snippets/x64/cpu_generator.cpp

```diff
@@ -287,6 +287,10 @@ size_t intel_cpu::CPUTargetMachine::get_lanes() const {
     }
 }
 
+size_t intel_cpu::CPUTargetMachine::get_reg_count() const {
+    return 16;
```

Comment:
It would be great if you create the ticket from [there](https://github.com/openvinotoolkit/openvino/pull/23180/files#r1574144699) and leave the comment with ticket number here.

Reply:
Ticket created and comment added. Thanks Alexandra!